### ISSUE TITLE
Kept getting "nvidia container" issues and got error for using image downloader sh file

### ISSUE THREAD

**Issue by Proryanator** [ogukei/easy-lora#1]
For windows, I have WSL2 and a 3080ti, and docker desktop.

Super interested in a quick lora training tool like this but kept getting an error that "nvidia" was not found when running the training (both in WSL2 and cmd).

Plus the .sh download file gave an error too.

Were you able to run this on Windows?

**Comment by ogukei** [comment]
Hello. 

> "nvidia" was not found when running the training

I was able to reproduce the problem on WSL 2 and have fixed it on PR #2.
Could you please try running it again on the latest main branch?

To ensure that you can access the GPU within the Docker under the WSL 2 environment, please run the following command:
```
docker run -it --rm --gpus all ubuntu nvidia-smi
```

> Plus the .sh download file gave an error too.

Well, I suspect that the script should be run via Docker. 
Please run the following command on WSL 2. If some errors occur, could you provide me with more details about the error?

```
docker compose run download_images
```

**Comment by Proryanator** [comment]
Lemme try the latest version, thanks!
